tools/nnpackage_tool/model2nnpkg/README.md

```diff
@@ -9,16 +9,16 @@ It takes `modelfile` as input and generates `nnpackage`.
 ```
 usage:  model2nnpkg.py [options]
   Examples:
-      model2nnpkg.py -m add.tflite                           => create nnpkg "add" in current directory
-      model2nnpkg.py -o out -m add.tflite                    => create nnpkg "add" in out/
-      model2nnpkg.py -o out -p addpkg -m add.tflite          => create nnpkg "addpkg" in out/
-      model2nnpkg.py -c add.cfg -m add.tflite                => create nnpkg "add" with add.cfg
-      model2nnpkg.py -o out -p addpkg -m a1.tflite a2.tflite => create nnpkg "addpkg" with models a1.tflite and a2.tflite in out/
+      model2nnpkg.py -m add.tflite                                              => create nnpkg "add" in current directory
+      model2nnpkg.py -o out -m add.tflite                                       => create nnpkg "add" in out/
+      model2nnpkg.py -o out -p addpkg -m add.tflite                             => create nnpkg "addpkg" in out/
+      model2nnpkg.py -c add.cfg -m add.tflite                                   => create nnpkg "add" with add.cfg
+      model2nnpkg.py -o out -p addpkg -m a1.tflite a2.tflite -i a1.json a2.json => create nnpkg "addpkg" with models a1.tflite and a2.tflite in out/
 
 
 Convert model files (tflite, circle or tvn) to nnpkg.
 
-options:
+optional arguments:
```

Comment:
What about keeping the term _option_? Maybe additional information of optional needs more changes. For example, `-i` is not required for simple single model.

Reply:
This is not a term I modified manually but python's `argparse` prints the term. Even so, I can restore it manually if you want.